app/Filament/Resources/CaseStudyResource.php

```diff
@@ -127,4 +131,13 @@ protected static function getTemplateOptions(): array
 
         return $options;
     }
+
+    protected static function getRequiredLocales(\Closure $data): array
```

Comment:
Lets import `Closure`

Reply:
Actually I forgot to remove this, it's not used so I deleted it